package.json

```diff
@@ -417,6 +427,11 @@
           "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions|localFunctionsEnablement)$/",
           "group": "c1@3"
         },
+        {
+          "command": "function.OpenInBrowserAction",
+          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localDeployFunctions)$/",
```

Comment:
This should be `viewItem =~ /^(localDeployFunctions|deployFunctions)$/` otherwise it doesn't work if you have a cluster only function.